tests/pass/libc.rs

```diff
@@ -5,8 +5,6 @@
 use std::fs::{remove_file, File};
 use std::os::unix::io::AsRawFd;
 
-extern crate libc;
```

Comment:
Does removing this mean that `./miri run` will no longer work on this file?

Reply:
If that is the only issue, I can add a `--run` or `--verbose` flag to the test suite.

Reply:
Or maybe we can just dump the original stderr [here](https://github.com/rust-lang/miri/blob/8757d866aba7f89ed0db5ae9f0b52be4715348d3/ui_test/src/lib.rs#L229)?

I think I am okay with those tokio/rand tests not working via `./miri run`, but the libc ones can easily work by using the libc from the sysroot so it's kind of a shame to lose that... would this still work if we kept the `extern crate` or does this just no longer work now?

Reply:
I can make the libc work again by adding the appropriate flag in `./miri run` directly

Reply:
So I take it currently it wouldn't work to just keep the `extern crate`? I could probably learn to live with this but it is kind of a bummer.

What would that look like? I thought those rmeta files have some pretty much random filename.

Reply:
well... *coughs* I can just do the ui_test thing and use the actual test `Cargo.toml` 😆

Reply:
I am confused by this reply...?

Reply:
What I mean is that I can just make `./miri run` use the `test_dependencies/Cargo.toml` together with `cargo-miri` to execute whatever file was passed, instead of passing it directly to `miri`. So basically what the test suite is doing already for each test